.github/workflows/codeql-analysis.yml

```diff
@@ -0,0 +1,50 @@
+name: "CodeQL"
+
+on:
+  push:
+    branches-ignore:
+      - 'dependabot/**'
```

Comment:
What does it do?

Reply:
I think it was added in the EDC repo to prevent runs on that branch. I don't think it is relevant for this repo, so I have removed it.